configstack/running_module_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 )
 
-var mockOptions = options.NewTerragruntOptionsForTest("running_module_test")
+var mockOptions, _ = options.NewTerragruntOptionsForTest("running_module_test")
```

Comment:
I would not ignore this error. It'll lead to confusing test failures. This may have to be converted to a `mockOptions(t *testing.T)` function that returns a `TerragruntOptions` struct...